sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubSchemaCapableIOProvider.java

```diff
@@ -155,16 +159,17 @@ private void validateConfigurationSchema(Row configuration) {
   /** An abstraction to create schema aware IOs. */
   @Internal
   private static class PubsubSchemaIO implements SchemaIO, Serializable {
```

Comment:
The `@Internal` flag is a warning to users to not depend on the labeled class. This class is not even visible to user because it is private, so the flag is not needed here.

Reply:
Done